src/averageAges.js

```diff
@@ -56,6 +84,28 @@ function calculateWomenAverageAge(people, withChildren) {
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
   // write code here
+  const womenMess = people.map(woman => woman.sex === 'f' ? woman : 0);
+
+  const women = womenMess.filter(woman => woman !== 0);
```

Comment:
combine it, you dont need map, use just filter

Reply:
it is done in new commit )